portfolio/src/main/webapp/script.js

```diff
@@ -28,3 +28,9 @@ function addRandomQuote() {
   const quoteContainer = document.getElementById('quote-container');
   quoteContainer.innerText = quote;
 }
+
+function updateGreeting() {
+  fetch('/data').then(response => response.text()).then((quote) => {
+    document.getElementById('quote-container').innerHTML = quote;
+  });
```

Comment:
nit: This might be more readable:
```
fetch('/data')
    .then(response => response.text())
    .then((quote) => {
        document.getElementById('quote-container').innerHTML = quote;
     });
```

Reply:
Updated the code. Thanks!